dispatch/tests/test_api_imagegallery.py

```diff
@@ -0,0 +1,370 @@
+from django.core.urlresolvers import reverse
+
+from rest_framework import status
+
+from dispatch.tests.cases import DispatchAPITestCase, DispatchMediaTestMixin
+from dispatch.apps.content.models import ImageGallery
+
+class ImageGalleryTests(DispatchAPITestCase, DispatchMediaTestMixin):
+    def _upload_image(self):
+        """
+        Upload an image that can be linked by galleries
+        """
+        url = reverse('api-images-list')
+
+        with open(self.get_input_file('test_image.png'), 'rb') as test_image:
+            response = self.client.post(url, { 'img': test_image }, format='multipart')
+
+        return response.data['id']
+
+    def _create_gallery_only(self, id, attachment):
```

Comment:
Maybe change `attachment` to `attachments` just to imply that there can be multiple